content/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Overview"
+date: 2021-11-16T12:08:47-05:00
+draft: false
+---
+
+# What is crane?
+
+Crane is a tool that helps application owners migrate kubernetes workloads and their state between clusters, remove environment-specific configuration and automate application deployments along the way. It is a tool that is divided into a few parts.
+
+- [crane](https://github.com/konveyor/crane): This is the cli(front end) of the process which allows user perform every action required.
+- [crane-lib](https://github.com/konveyor/crane-lib): This is the brains behind crane actions and is responsible for transforming resources.
+- [crane-plugin-openshift](https://github.com/konveyor/crane-plugin-openshift): This is a custom plugin that can be added as an extension to the `transform` command to have additional actions as per need while transforming resources. 
+- [crane-plugins](https://github.com/konveyor/crane-plugins): This is the central source of truth from where any plugins can be added to the migration process with the help of plugin-manager.
+- Additional plugins will be added as new "crane-plugin-<plugin>" repos
+
+> Cli commands and plugins are explained in depth in later part of the document, visit Usage to understand how crane works.
+
+# Why crane is needed?
+
+Crane is a pipelined tool unlike `MTC` where now using crane user can intervene after each step (i.e `export` and `transform`) to make sure that the process done up till current step is valid and useful for the migration. Another advantage of `crane` tool is that it provides flexibility to add plugins (custom logic to the pipeline to cater a specific need) while transforming resources for destination cluster.
```

Comment:
```suggestion
Crane is the product of our team distilling several years of experience performing
large-scale production Kubernetes migrations. These operations are large, complex,
error-prone, and usually must be peformed under a limited window of time. Because
of that challenge, its paramount that a migration tool be designed with transparency
and ease-of-diagnostics in mind. Crane is designed to drive a migration via a
pipeline of non-destructive tasks that dump their results to disk so the operation
can be easily audited and versioned without ever impacting live workloads.
The tasks are [idempotent](https://en.wikipedia.org/wiki/Idempotence), meaning
they can be run repeatedly and will output consistent results given the same
inputs without side-effects on the system at large.
```